charts/dave/Chart.yaml

```
@@ -5,34 +5,34 @@ type: application
 version: 0.0.12
 appVersion: "v1.0.0"
 maintainers:
-  - name: gislab-augsburg
-    email: martymcfly333@gmail.com
-  - name: klml
-    email: klml@muenchen.de
+- name: gislab-augsburg
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 1

(indentation)